cg/store/filters/status_order_filters.py

```diff
@@ -42,6 +42,10 @@ def filter_orders_by_search(orders: Query, search: str | None, **kwargs) -> Quer
     )
 
 
+def filter_orders_by_delivered(orders: Query, delivered: bool | None, **kwargs) -> Query:
+    return orders.filter(Order.delivered == delivered) if delivered is not None else orders
```

Comment:
Jus a question, is the None due to the use of a pydantic model when passing parameters? I think if delivered is not supplied to the `apply_delivered_filter` then this function would do nothing, right?

Reply:
The flow goes smoother in this case since we can remove a lot of if-statements depending on what the HTTP request looks like. With this design, we can fetch orders via the /orders endpoint and regardless of what parameters are set (pagination, search, sort order, order_id etc), we can specify "apply all filters" and only the filters with the parameter set in the request will actually get invoked.

Reply:
Yes, makes sense.